README.md

```diff
@@ -29,9 +29,10 @@ After installation, you can add a widget to your app. Here's a simple [button ex
 ```
 
 ```js
-$(function () {
-    $("#buttonContainer").dxButton();
-});
+import Button from 'devextreme/ui/button';
+
+let element = document.getElementById('buttonContainer');
+let button = new Button(element, { text: 'Hello World!' });
```

Comment:
Seems, we can simplify to:
let button = new Button("#buttonContainer", { text: 'Hello World!' });

Reply:
This way, I have no idea if it's based on jQuery or not. I would keep the most transparent example when the Button accepts a pure JS element.

Reply:
`const` 😄

Reply:
`new Button("#buttonContainer")`
It will work with jQuery and without it as well.